Makefile

```diff
@@ -25,9 +25,15 @@ integration:
 
 test-docker:
 	$(DOCKER_RUN) \
-	  --volume "$(PWD)":"/src/$(PROJECT):ro" \
+	  --volume "$(PWD)":"/src/$(PROJECT):rw" \
```

Comment:
I think the default is `rw` so if we no longer want this to be a read-only volume mount, we could remove the suffix altogether.